frontend/test/jest-setup.js

```diff
@@ -1,4 +1,3 @@
-import { XMLHttpRequest } from "xmlhttprequest";
```

Comment:
This was introduced in https://github.com/metabase/metabase/pull/27337, but doesn't seem to be needed to work in Node v16+ anymore